R/extract.R

```diff
@@ -27,11 +27,15 @@
 #'
 #' - `extract_parameter_set_dials()` returns a set of dials parameter objects.
 #'
+#' - `extract_fit_time()` returns a tibble with elapsed fit times.
```

Comment:
```suggestion
#' - `extract_fit_time()` returns a tibble with elapsed fit times. The fit times correspond to
#'   the time for the parsnip engine or recipe steps to fit (or their sum if `summarize = TRUE`)
#'   and do not include other portions of the elapsed time in [workflows::fit.workflow()].
```

Reply:
Will need a re`document()` and confirmation that that link heads to the right help-page!